next-app/src/routes/password/change/+page.server.ts

```diff
@@ -0,0 +1,33 @@
+import { sf, type Rpc } from '$lib/server/sf'
+import { fetch_current_user } from '$lib/server/user'
+import { fail, redirect } from '@sveltejs/kit'
+import type { RequestEvent } from '../$types'
+
+export const actions = {
+  default: async ({ request }: RequestEvent) => {
+	const submissions = await request.formData()
```

Comment:
[nit] Looks like this function-body should be indented

Reply:
thanks, [0f3b636](https://github.com/sillsdev/web-languageforge/pull/1716/commits/0f3b63692394cccd007966ec38d26bf878a03a28)